### DISCUSSION TITLE
Buttons not responds properly

### DISCUSSION THREAD

**Discussion by Marco-Buonaguidi** [nefarius/DsHidMini#96] [Q&A]
BthPS3 1.3.127.0
Dshidmini v1.4.222.0
PCSX" ver. 1.6.0

I have followed the installation guide and i've tried to use the PS3 controller on PCSX2. The mapping of the buttons on PCSX2 work fine and all buttons are correctly recognized and mapped, but when i start a game the buttons doesn't respond properly. For example, the start button doesn't work but if i press L2 button it work like start button and L2 button at same time, as if both inputs are sent to the emulator. Accordingly to dshidmini, my controller is genuine.

**Comment by Kanuan** [comment]
1. Send a screenshot of the DsHidMinic Control Cliente (DSHMC)
2. Is your controller in SDF mode?
3. Check the [_How do I set the controller's pressure buttons in PCSX2?_](https://vigem.org/projects/DsHidMini/FAQ/#how-do-i-set-the-controllers-pressure-buttons-in-pcsx2) topic in the FAQ

## EDIT 1

When changing modes, sometimes the `apply settings` button does not work right away, meaning the controller has not changed modes even though DSHMC says otherwise. To be 100% sure it changed to the new mode, reconnect the controller (unplug/replug on USB, disconnect and reconnect on Bluetooth)

**Comment by Marco-Buonaguidi** [comment]
Hi,

i've attached the requested screenshots. I've followed the guide and i've set the SDF mode and used the "Mute digital pressure buttons" to configure the pressure sensitive buttons. The problem persist, even if i connect the pad via usb, there isn't any difference.
If can help, the controller that i'm using is a SIXAXIS, not a DUALSHOCK 3. Is the first version of the controller, that was without vibration.
I also have another controller DUALSHOCK 3 that is reported as fake and it has the same problems.

Let me know if you have necessity of other information to debug the problem, meanwhile i'll do other tests with other versions of PCSX2 too.

________________________________
Da: Kanuan ***@***.***>
Inviato: mercoledì 19 maggio 2021 01:09
A: ViGEm/DsHidMini ***@***.***>
Cc: Marco Buonaguidi ***@***.***>; Author ***@***.***>
Oggetto: Re: [ViGEm/DsHidMini] Buttons not responds properly (#89)


  1.  Send a screenshot of the DsHidMinic Control Cliente (DSHMC)
  2.  Is your controller in SDF mode?
  3.  Check the How do I set the controller's pressure buttons in PCSX2?<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fvigem.org%2Fprojects%2FDsHidMini%2FFAQ%2F%23how-do-i-set-the-controllers-pressure-buttons-in-pcsx2&data=04%7C01%7C%7C2ed0dfe46e3946fa23e908d91a51fb13%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637569761665869887%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=GQ6JfjXa9jIAy615C%2Bwcgj68q6OBj2%2BpCL8ynyWiJIs%3D&reserved=0> topic in the FAQ

**Comment by nefarius** [comment]
I'm afraid GitHub strips away any attachments if you reply via mail rather than the Web browser so I'm afraid we can't see what you see 😊

**Comment by Marco-Buonaguidi** [comment]
Sorry

![DsHidMini](https://user-images.githubusercontent.com/24478265/119028112-629c0080-b9a7-11eb-84ba-6327314a020d.PNG)
![PCSX2-1](https://user-images.githubusercontent.com/24478265/119028115-63349700-b9a7-11eb-883a-8318b28a24c5.PNG)
![PCSX2-2](https://user-images.githubusercontent.com/24478265/119028117-63349700-b9a7-11eb-9aca-86061b810a41.PNG)

**Comment by Marco-Buonaguidi** [comment]
Hi guys,

after other tests i can say that the problem occur only with some games. I've tried to play with Gran Turismo 3 and Gran Turismo 4 and both have the problem, but i can play Metal Gear Solid 2 normally, the problem doesn't occur on this game.

**Comment by nefarius** [comment]
> Hi guys,
> 
> after other tests i can say that the problem occur only with some games. I've tried to play with Gran Turismo 3 and Gran Turismo 4 and both have the problem, but i can play Metal Gear Solid 2 normally, the problem doesn't occur on this game.

Just to clarify; we're still speaking exclusively about games running PCSX2, correct?

**Comment by Marco-Buonaguidi** [comment]
Yes, i've used the pad only with PCSX2 at the moment.

**Comment by nefarius** [comment]
How odd, once the mappings are set up, I see no logic in why games would respond differently to the emulator configuration...

**Comment by Marco-Buonaguidi** [comment]
I don't know if could help, but i have a similar problem with a Logitach G923 racing wheel. During the configuration of the input via the plugin on PCSX2 (USBqemu-wheel.dll plugin) all seems to work properly, but in the game (gran turismo 3) the inputs are wrong, like the problem with the pad. Could be a problem of PCSX2?

However, could be the problem reported in this issue the same reported in #68 ?

**Reply by nefarius** [reply]
> However, could be the problem reported in this issue the same reported in #68 ?

No, that is a different topic.

**Comment by nefarius** [comment]
Guess I need to take a look at the LilyPad sources to understand what's going on.

**Comment by Marco-Buonaguidi** [comment]
Hi guys, i'm not sure if it's a bug of the DS3 driver or PCSX2, but the problem seems to be related to the presence of the USBqemu-wheel plugin; if i remove this plugin the controller work properly!

**Reply by nefarius** [reply]
No idea what version of PCSX2 you're on or your config or the plugin you mentioned... And please don't unearth age old discussions and just make your own, thanks 👍

**Comment by prataczak** [comment]
I'm having this issue as well. It only occurs while plugged into the USB port, when wireless, everything is fine. Also vibration does not work while plugged in. This is while in 'SDF' mode. Changing to Xinput in the DSHidmini control, there aren't any issues with the start button while plugged in.
